OSDOCS-10828: 4.15.17 RN (#77297)
openshift/openshift-docs: release_notes/ocp-4-15-release-notes.adoc

For any {product-title} release, always review the instructions on xref:../updating/updating_a_cluster/updating-cluster-web-console.adoc#updating-cluster-web-console[updating your cluster] properly.
====

[id="ocp-4-15-17"]
=== RHBA-2024:3673 - {product-title} 4.15.17 bug fix update and security update

Issued: 2024-06-11

> {product-title} release 4.15.17 is now available. The list of bug fixes that are included in this update is documented in the link:https://access.redhat.com/errata/RHBA-2024:3673[RHBA-2024:3488] advisory. The RPM packages that are included in this update are provided by the link:https://access.redhat.com/errata/RHSA-2024:3676[RHSA-2024:3676] advisory.

Author: good catch! ty!

[id="ocp-4-15-17-bug-fixes"]
==== Bug fixes

* Previously, when multiple domains were configured for an Amazon Virtual Private Cloud (VPC) DHCP option, the hostname could return multiple values. The logic did not account for multiple values, and crashed when it returned a node name containing a space. with this release, the logic has been updated to use only the first returned hostname as the node name and the issue has been resolved. (link:https://issues.redhat.com/browse/OCPBUGS-33847[*OCPBUGS-33847*])

* Previously, when enabling `virtualHostedStyle` with `regionEndpoint` set in the image registry operator configuration, the image registry ignored the virtual hosted style configuration and failed to start. With this release, the image resistry uses a new upstream distribution configuration and the issue has been resolved. (link:https://issues.redhat.com/browse/OCPBUGS-34539[*OCPBUGS-34539*])

> * Previously, OperatorHub incorrectly excluded the Amazon Resource Name (ARN) role information for ROSA Hosted Control Plane (HCP) clusters. With this update, OperatorHub correctly displays ARN information and the issue has been resolved. (link:https://issues.redhat.com/browse/OCPBUGS-34550[*OCPBUGS-34550*])

Reviewer: ```suggestion
* Previously, the OperatorHub incorrectly excluded the Amazon Resource Name (ARN) role information for ROSA Hosted Control Plane (HCP) clusters. With this update, the OperatorHub correctly displays ARN information and the issue has been resolved. (link:https://issues.redhat.com/browse/OCPBUGS-34550[*OCPBUGS-34550*])
```

> * Previously, when attempting to delete a cluster or BareMetalHost (BMH) before installation, the metal3-operator tried to unnecessarily generate a pre-provisioning image. With this release, an exception has been created to prevent the creation of a pre-provisioning image during a BMH deletion and the issue has been resolved. (link:https://issues.redhat.com/browse/OCPBUGS-34682[*OCPBUGS-34682*])

Reviewer: ```suggestion
* Previously, when attempting to delete a cluster or BareMetalHost (BMH) resource before installation, the metal3-operator tried to unnecessarily generate a pre-provisioning image. With this release, an exception has been created to prevent the creation of a pre-provisioning image during a BMH deletion and the issue has been resolved. (link:https://issues.redhat.com/browse/OCPBUGS-34682[*OCPBUGS-34682*])
```

> * Previously, select text areas were no longer resizeable when editing a config map in the *Form View* of the web console in {product-title} 4.15. With this release, those text areas are now resizeable. (link:https://issues.redhat.com/browse/OCPBUGS-34703[*OCPBUGS-34703*])

Reviewer: ```suggestion
* Previously, some text areas were no longer resizable when editing a config map in the *Form View* of the web console in {product-title} 4.15. With this release, those text areas are now resizable. (link:https://issues.redhat.com/browse/OCPBUGS-34703[*OCPBUGS-34703*])
```
